tests/utils.go

```diff
@@ -1694,8 +1695,47 @@ func removeNamespaces() {
 	fmt.Println("")
 	for _, namespace := range testNamespaces {
 		fmt.Printf("Waiting for namespace %s to be removed, this can take a while ...\n", namespace)
-		EventuallyWithOffset(1, func() bool { return errors.IsNotFound(virtCli.CoreV1().Namespaces().Delete(namespace, nil)) }, 240*time.Second, 1*time.Second).
-			Should(BeTrue())
+		EventuallyWithOffset(1, func() bool {
+			namespaceDeletionError := virtCli.CoreV1().Namespaces().Delete(namespace, nil)
+			if namespaceDeletionError != nil {
+				fmt.Printf("could not get delete '%v':\n %v\n", namespace, namespaceDeletionError)
+			}
+
+			namespaceObject, err := virtCli.CoreV1().Namespaces().Get(namespace, metav1.GetOptions{})
+			if err != nil {
+				fmt.Printf("could not get namespace '%v':\n %v\n", namespace, err)
+			}
+
+			fmt.Printf("deleting namespace '%v':\n %v\n", namespace, err)
```

Comment:
why `err` is printed here as well? it could the error from the GET operation which is not related to `deleting namespace`?